### DISCUSSION TITLE
Change history

### DISCUSSION THREAD

**Discussion by khaitranhq** [keystonejs/keystone#5998] [Feature requests]
I'm using keystone like CMS for my project. A few days ago someone in my team edited data but I can't find who he/she is. I think it's better if Keystone has a function to manage the change history of data.
Thanks

**Comment by raveling** [comment]
@lioaslan we asked the community about version control and history for list items in a recent survey, and many other features were preferred above this. 

It's definitely an idea we are aware of, but are unlikely to pursue it in the next 6–12 months at least.

Hope this helps!
